api/valor_api/schemas/status.py

```diff
@@ -1,6 +1,19 @@
 from pydantic import BaseModel
 
 
+class APIVersion(BaseModel):
+    """
+    Defines an API version string which is sent back to the user after their authentication is confirmed.
```

Comment:
nit: out of curiosity, did you find a bug that prompted the need for this schema?

Reply:
This existed prior to this pr alone in another file.

Reply:
Ill revert this.